pkgs/by-name/pl/playball/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "playball";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "paaatrick";
+    repo = "playball";
+    rev = "v${version}";
+    hash = "sha256-xgAhzNWCLNmbrwaYAGmXMercoRgXWPjjV5dcnXunmeA=";
+  };
+
+  npmDepsHash = "sha256-s0JKBJnVYkeXOE62F6BZRKwd0Hg3IOuMai6rmKUi6TI=";
+
+  env.ELECTRON_SKIP_BINARY_DOWNLOAD = "1";
+
+  meta = with lib; {
+    changelog = "https://github.com/paaatrick/playball/releases/tag/v${version}";
+    description = "Watch MLB games from the comfort of your own terminal";
+    mainProgram = "playball";
+    homepage = "https://github.com/paaatrick/playball";
+    license = licenses.mit;
+    maintainers = with maintainers; [ nickthegroot ];
+  };
```

```suggestion
  meta = {
    changelog = "https://github.com/paaatrick/playball/releases/tag/v${version}";
    description = "Watch MLB games from the comfort of your own terminal";
    mainProgram = "playball";
    homepage = "https://github.com/paaatrick/playball";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ nickthegroot ];
  };
```
`with` before attrsets (especially `with lib;`, _especially_ file-wide ones, _**especially**_ multiple ones) has been abused to death in the past and is now frowned upon. Just prepend `lib.` where necessary, it's just 4 chars